content/v2/openapi.yml

```
@@ -1838,6 +1856,57 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
       summary: Check zone record distribution
+  '/{account}/zones/{zone}/resolution':
+    put:
+      description: |-
+        Enables DNS resolution for the zone.
```

I could be me, but given they're turning on a paid service, the combo Activate/Deactivate seems more convincing to me than Enable/Disable.

```suggestion
        Activate DNS services for the zone.
```